src/Files.App/Constants.cs

```diff
@@ -240,6 +240,15 @@ public static class UserEnvironmentPaths
 
 			public static readonly string RecentItemsPath = Environment.GetFolderPath(Environment.SpecialFolder.Recent);
 
+			public static string SystemDrivePath
+			{
+				get
+				{
+					var systemDrive = Environment.GetEnvironmentVariable("SystemDrive");
+					return !string.IsNullOrEmpty(systemDrive) ? systemDrive : "C:";
+				}
+			}
```

Comment:
```suggestion
			public static readonly string SystemDrivePath;

			static UserEnvironmentPaths()
			{
				var systemDrive = Environment.GetEnvironmentVariable("SystemDrive");
				SystemDrivePath = !string.IsNullOrEmpty(systemDrive) ? systemDrive : "C:";
			}
```
The static constructor allows it to remain a readonly field.